File: Assignment/taxcalculator.py
for numbers in range( 1, 3, 1): 
	name = input("Enter your name: ")
	earnings = int(input("Enter earnings: "))
	if(earnings <= 30000):
		taxrate = 15/100 * earnings
		print (taxrate)
	if (earnings > 30000):
		taxrate = 20/100 * earnings;
		print (taxrate)

File: Assignment/java tasks/TaxCalculator.java
import java.util.Scanner;
public class TaxCalculator{

public static void main(String... args){

Scanner scanner = new Scanner(System.in);

for(int count = 1; count<=3; count++)

System.out.print("Enter your name");
name = scanner.nextLine();

System.out.print("Enter earnings");
int earnings = scanner.nextInt();


if(earning <= 30000){
taxRate = (15/100) * earning;
System.out.println(taxrate);
}

else {
if(earning > 30000);
taxRate = (20/100) * earning;
System.out.println(taxrate);

}







}
}